test/e2e/ssm/command.go

```diff
@@ -8,12 +8,18 @@ import (
 
 	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/service/ssm"
+	"github.com/aws/aws-sdk-go-v2/service/ssm/types"
+	smithytime "github.com/aws/smithy-go/time"
 	"github.com/go-logr/logr"
 
 	e2eCommands "github.com/aws/eks-hybrid/test/e2e/commands"
 )
 
-const commandTimeout = 10 * time.Minute
+const (
+	commandTimeout               = 10 * time.Minute
+	instanceRegisterTimeout      = 5 * time.Minute
+	instanceRegisterSleepTimeout = 15 * time.Second
```

Comment:
is this a random number?
or do instances usually take around 15s to register?

Reply:
its just random. that is the sleep time if the instance is not register, before it retries the describe.  the overall timeout of 5 mins *should* be plenty, for the jumpbox which we are using this for, its usually up and registered before we ever even get here.